R/config.R

```diff
@@ -234,8 +234,8 @@ python_environment_versions <- function() {
       workon_home <- NULL
     # all possible environment dirs
     env_dirs <- c(workon_home, "~/.virtualenvs", 
-                  "~/anaconda/envs", "~/anaconda3/envs", "~/anaconda3/envs", 
-                  "/anaconda/envs", "/anaconda3/envs", 
+                  "~/anaconda/envs", "~/anaconda3/envs", "~/anaconda3/envs", "~/miniconda/envs", "~/miniconda3/envs",
+                  "/anaconda/envs", "/anaconda3/envs", "miniconda/envs", "miniconda3/envs",
```

Comment:
This needs to be a full absolute path I think? (e.g. `/miniconda/envs`)

Reply:
Could you also a note to NEWS?

Reply:
Oops. Done!